src/pages/IndexPage.vue

```diff
@@ -89,14 +92,15 @@
 import { defineComponent } from 'vue';
 
 // import components
-import EventCountdown from 'components/EventCountdown.vue';
-import ListCardChallenge from 'components/ListCardChallenge.vue';
+import BannerApp from 'src/components/BannerApp.vue';
 import BannerImage from 'components/BannerImage.vue';
```

Comment:
The Quasar framework allows you to use [path alias to import modules](https://quasar.dev/quasar-cli-webpack/handling-webpack#webpack-aliases). 

Use same **path alias** (components) import for all modules e.g. 

```ts
import ListCardEvent from 'components/ListCardEvent.vue';
```

instead of 

```ts
import ListCardEvent from 'src/components/ListCardEvent.vue';
```
please.

Reply:
Fixed. I will update the import paths across the repo in a separate PR.